consensus/ibft/consensus_backend.go

```diff
@@ -139,39 +140,21 @@ func (i *backendIBFT) MaximumFaultyNodes() uint64 {
 	return uint64(CalcMaxFaultyNodes(i.currentValidators))
 }
 
-func (i *backendIBFT) HasQuorum(
-	blockNumber uint64,
-	messages []*proto.Message,
-	msgType proto.MessageType,
-) bool {
-	validators, err := i.forkManager.GetValidators(blockNumber)
-	if err != nil {
-		i.logger.Error(
-			"failed to get validators when calculation quorum",
-			"height", blockNumber,
-			"err", err,
-		)
+func (i *backendIBFT) GetVotingPowers(height uint64) (map[string]*big.Int, error) {
```

Comment:
Could you add a disclaimer that this will be deprecated, so that is why we are just returning 1 as voting power of validators?

Reply:
fixed: https://github.com/0xPolygon/polygon-edge/pull/1392/commits/c8b3d8df284c829644f87d9a2f3345d0f7d0d23d